### PR TITLE
Two new modules, and some minor mapper improvements

### PR DESCRIPTION
**Pull request by silasary**

### CODE REVIEW

File: modules/file_editor.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import os
+
+from modules.basemodule import BaseModule
+
+
+class FileEdit(BaseModule):
+    """
+    Use your mud client's builtin editor to modify files.
+
+    This is only really useful if pycat isn't running on the same machine as your client.
```
Comment (cizra) [inline modules/file_editor.py:10]: May I ask, why is this useful at all? Isn't it more straightforward to SSH into the box with pycat, and use an actual editor like Vim or Emacs or Nano or something?

**Review by cizra — CHANGES_REQUESTED** [review]

File: modules/file_editor.py (hunk 1 above)
Comment (cizra) [reply modules/file_editor.py:10]: The rest of the PR I agree with, by the way. Flattered that somebody's using it.

Comment (silasary) [reply modules/file_editor.py:10]: I have it running in a docker container in a kubernetes cluster. The actual files are in a Persistent Volume, and are a lot harder to access than you'd think

Comment (silasary) [reply modules/file_editor.py:10]: Besides, my mud client has a built in editor, and typing `#edit triggers.json` is always going to be more convenient than pulling up a terminal and doing stuff over ssh

Comment (cizra) [reply modules/file_editor.py:10]: Right. Editing Docker volumes is a pain indeed.

**Pull request merged**